pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<!--
+If your PR resolves or is related to one or multiple issues, please mention so here using "Closes #ISSUE_NUMBER" or "Related to #ISSUE_NUMBER", respectively, for each issue.
```

Comment:
```suggestion
If your PR resolves one or more issues, please mention so here using "Closes #ISSUE_NUMBER", for each issue. This will hook things up so that those issues are automatically closed when the PR is merged.

If your PR is related to (but does not fully fix) one or more issues, please mention so here using "Related to #ISSUE_NUMBER" or "Towards #ISSUE_NUMBER", for each issue.
```